src/components/Button/Button.tsx

```diff
@@ -15,26 +15,26 @@ export interface ButtonProps extends ComponentPropsWithoutRef<"button"> {
 export function Button(props: ButtonProps) {
   const { children, preIcon, posIcon, variant, ...rest } = props;
 
-  const variantClass = variant
-    ? "text-f-primary " +
-      "hover:bg-neutral/[.15] " +
-      "active:bg-neutral/[.3] " +
-      "disabled:text-f-disabled-1 disabled:bg-c-disabled-2 disabled:border-c-disabled-1"
-    : "bg-c-default text-primary " +
-      "hover:bg-c-hover " +
-      "active:bg-c-pressed " +
-      "disabled:text-f-disabled-2 disabled:bg-c-disabled-1 disabled:border-c-disabled-1";
+  const primaryClass = `bg-primary text-f-primary
+                        hover:bg-tertiary 
+                        active:bg-secondary 
```

Comment:
Why did you remove `bg-neutral/[.15]` ?

Reply:
Because `[.15]` it's for the opacity. On the model, `hover` and `active` states have an opacity.

Reply:
Ah sorry!
I just saw the 2nd class below